unstructured_inference/yolox_functions.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Megvii, Inc. and its affiliates.
```

Comment:
These functions are only used in `yolox_model.py`. Can we move them there to keep all the `yolox` code in one place?